app/js/main.js

```diff
@@ -229,6 +229,9 @@ function startFlowAuto() {
       // Forget any subscriptions, for metering demo purposes.
       subscriptions.clear();
 
+      // Set up metering demo language.
+      document.documentElement.lang = getQueryParams().lang || 'en';
```

Comment:
You can get the browser default language or browser current language instead of just assuming english.  Unless that might break things due to an unsupported language appearing.

https://stackoverflow.com/questions/1043339/javascript-for-detecting-browser-language-preference

Reply:
The Scenic articles are written in English, so I think English is a good default here